backend/validators/auth.api.validator.js

```diff
@@ -0,0 +1,42 @@
+const { body, validationResult } = require('express-validator');
```

Comment:
What about file name `user.validator.js` instead of `auth.api.validator.js`? Looks like it validates not only auth but creating user calls too.